build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+
+    repositories {
+        google()
+        mavenCentral()
+        maven(url = "https://plugins.gradle.org/m2/")
+    }
+    dependencies {
+        classpath("com.android.tools.build:gradle:7.2.1")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.21")
+        classpath("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.19.0")
+        classpath("com.google.dagger:hilt-android-gradle-plugin:2.45")
+        classpath("com.github.ben-manes:gradle-versions-plugin:0.29.0")
+        classpath("org.jetbrains.kotlinx:kover:0.4.1")
+        classpath("com.hiya:jacoco-android:0.2")
+        classpath("gradle.plugin.org.kt3k.gradle.plugin:coveralls-gradle-plugin:2.12.0")
```

Comment:
Before we merge, let's move these into our toml file and define them properly, but we can continue to use hardcoded values here to get this compiling again.